### PR TITLE
Added formula 8.16 from NEN-EN 1992-1-1+C2:2011

### LINKED ISSUE #87: [✨ Feature request]: Add formula 8.16 from NEN-EN 1992-1-1+C2:2011
### Description

Add formula 8.16 from NEN-EN 1992-1-1+C2:2011

### Acceptance Criteria

Formula 8.16
Unittesting
Complete docstring

### Dependencies

None

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by egarciamendez**
## Description

Added formula 8.16 from NEN-EN 1992-1-1+C2:2011

Fixes #87 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        alpha_2 : DIMENSIONLESS
+            [:math:`α_{2}`] Coefficient taking account of the type of prestressing steel [-].
+
+            = 0.25 for tendons with circular cross-section;
+
+            = 0.19 for 3 and 7-wire strands.
+
+            Use your own implementation for this value or use :class:`SubForm8Dot16Alpha2` class.
+        diameter : DIMENSIONLESS
+            [:math:`Ø`] Nominal diameter of the tendon [:math:`mm`].
```
Comment (SZeltaat) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:46]: I'm not really familiar with the subject. This seems contradictory to me.
From what I can understand from the context, I think diameter is not DIMENSIONLESS here. I think the correct type would be MM.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        self.f_bpt = f_bpt
+
+    @staticmethod
+    def _evaluate(
+        alpha_1: DIMENSIONLESS,
+        alpha_2: DIMENSIONLESS,
+        diameter: DIMENSIONLESS,
+        sigma_pm0: MPA,
+        f_bpt: MPA,
+    ) -> MPA:
```
Comment (SZeltaat) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:68]: I believe this should be MM.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
+"""Formula 8.16 from NEN-EN 1992-1-1+C2:2011: Chapter 8: Detailing of reinforcement and prestressing tendons"""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import DIMENSIONLESS, MPA
```
Comment (SZeltaat) [inline blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:5]:
```suggestion
from blueprints.type_alias import DIMENSIONLESS, MPA, MM
```

**Review by SZeltaat — COMMENTED** [review]
Good work Enrique!
I just have some comments on typehints. Good luck! 😉

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py (hunk 1 above)
Comment (egarciamendez) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:46]: That's correct! It is a 'typo' from my side.

File: blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py (hunk 2 above)
Comment (egarciamendez) [reply blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_16.py:68]: Indeed it should.

**Comment by egarciamendez** [comment]
> Good work Enrique! I just have some comments on typehints. Good luck! 😉

Fixed! thanks for the review 😄

**Pull request merged**